kafka-ui-react-app/src/theme/theme.ts

```diff
@@ -40,6 +41,12 @@ export const Colors = {
     '30': '#90caf9',
     '40': '#64b5f6',
   },
+  gray: {
+    '10': '#f9fafa',
+    '80': 'rgba(34, 41, 47, 0.5)',
+    '85': 'rgba(10, 10, 10, 0.1)',
+    '90': 'rgba(0, 0, 0, 0.1)',
+  },
```

Comment:
I think it should be a part of `neutral`

Reply:
Is`t `neutral` filled enough?

Reply:
Is`t neutral filled enough?

Reply:
agree with workshur